Sources/tart/Commands/Get.swift

```diff
@@ -0,0 +1,31 @@
+import ArgumentParser
+import Foundation
+
+struct Get: AsyncParsableCommand {
+  static var configuration = CommandConfiguration(commandName: "get", abstract: "Get a VM's configuration")
+
+  @Argument(help: "VM name")
+  var name: String
+
+  func run() async throws {
+    do {
+      let vmDir = try VMStorageLocal().open(name)
+      let vmConfig = try VMConfig(fromURL: vmDir.configURL)
+      let diskSize = try vmDir.sizeBytes() / 1024 / 1024 / 1000
+
+      print("cpu\tmemory\tdisk\tdisplay")
+
+      var s = "\(vmConfig.cpuCount)\t"
+      s += "\(vmConfig.memorySize / 1024 / 1024)MB\t"
```

Comment:
Added a spacing to improve readability:

```suggestion
      s += "\(vmConfig.memorySize / 1024 / 1024) MB\t"
```